tests/run/cppclass_overloading_dropped.pyx

```diff
@@ -0,0 +1,18 @@
+cdef:
+	extern from "cpp_source.h":
+		cppclass testclass:
+			testclass()
+			void a()
+			void a(unsigned int i)
+	cppclass testclass2(testclass): # a() is dropped
```

Comment:
This issue here is that

https://github.com/cython/cython/blob/0ab91a54bd2c54abf9c09a60cdf10bc8a271219a/Cython/Compiler/Symtab.py#L2576

doesn't copy `overloaded_alternatives`properly. Actually fixing it looks slightly fiddly (and I've run out of time at the moment)